.github/workflows/compile-queries.yml

```diff
@@ -24,21 +24,21 @@ jobs:
         run: |
           MERGE_BASE=$(git merge-base --fork-point origin/$BASE_BRANCH)
```

Comment:
We have some code in the Foundations team benchmarking infrastructure that does this by calling the GitHub API, you could do something similar here: https://github.com/github/semmle-code/blob/db3086b9699c8c8c9846cd6239b78da431f27dc3/.github/actions/benchmark-prepare/lib/index.js#L55-L70 (with apologies to any external users for the link to our internal code)

Reply:
This should work too. There must be a simpler way....
```
git cat-file commit  HEAD | grep '^parent ' | head -1 | cut -d ' ' -f 
```

Reply:
> This should work too. There must be a simpler way....

Reply:
The `HEAD` commit should be a merge commit of the the base and the head branches of a pull request.

Reply:
See also: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

Reply:
Yes, I know. But I still don't see how that's useful for getting the merge-base. 

I have something better when running on main (´github.sha`).  
And when running in a PR I don't see how I can use the HEAD commit, because the parent of the HEAD commit is most of the time not the commit I'm looking for.